aws/README.md

```diff
@@ -0,0 +1,54 @@
+
+# AWS Setup files and scripts
+
+Current directory contains files and scripts to setup environment for Hashicorp Vault, Postgres and Controller.
+
+## Setup Backend environment
+
+To setup Backend environment, we have to create a VPC in any region having 3 Public Subnets, 3 Private Subnets, 1 Internet Gateway, 1 NAT Gateway, 1 SSL/TLS certificate created/imported in AWS Certificate Manager (ACM)
+
+#### Steps:
+
+    1) Run following script which checks for cloudformation stack and VPC existance and creates resources accordingly:
+
+    ./create-backend-vpc-cloudformation-stack.sh
+
+## Create SSL Certificate (Before creating vault cloudformation stack)
+
+To create SSl certificate, which is needed while deploying vault, use following script, which checks if certificate is already created or not, and creates the certificate with domain name 'vault.ddx.drio.ai':
+
+    ./create-vault-ssl-certificate-cloudformation-stack.sh
+
+
+## Deploy Postgres RDS instance in Backend environment
+
+Following script to deploy Postgres RDS needs some environment pre-requisites:
+
+    i) VPC with 2 Subnets in different availability zones. Above "create-backend-vpc-cloudformation-stack.sh" script creates suitable environment needed to deploy Postgres RDS.
+    ii) Subnet Groupt with minimum 2 subnets in different availibility zone.
+
+Steps to deploy Postgres RDS:
+
+#### NOTE: USE ONLY ONE OF THE FOLLOWING WAYS TO CREATE BACKEND VPC
+
+    1.1) RECOMMENDED WAY: Modify the script and edit variable value "MASTER_USER_PASSWORD", modify other variables as well as per requirement.
```

Comment:
What is MASTER_USER_PASSWORD? Is it for AWS or Postgres?

Reply:
It is for postgres user. I will rename this variable to POSTGRES_MASTER_USER_PASSWORD